setup.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+    remove: [
+        { file: 'resources/icons/512x512.png' },
+        { file: 'resources/osx/dmg-background.png' },
+        { file: 'resources/osx/dmg-background@2x.png' },
+        { file: 'resources/osx/dmg-icon.icns' },
+        { file: 'resources/osx/icon.icns' },
+        { file: 'resources/windows/icon.ico' },
+        { file: 'resources/windows/setup-icon.ico' },
+        { folder: 'src/hello_world' }
```

Comment:
Ohhhh that's a good point!  Great idea, and totally agreed.  Perhaps I should bake that into the product... Either way, for now I'll add it explicitly.  Thanks! 👍  🚀